### ISSUE TITLE
Descriptors HowTo: Example on function.__get__ needs update

### ISSUE THREAD

**Issue by 749c80fe-1c97-4e46-b1dd-74bf62c51f66** [python/cpython#74715]
BPO | [30530](https://bugs.python.org/issue30530)
--- | :---
Nosy | @rhettinger, @Mariatta, @rmariano
PRs | <li>python/cpython#1845</li><li>python/cpython#1953</li><li>python/cpython#1954</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = 'https://github.com/Mariatta'
closed_at = <Date 2017-06-05.03:10:35.444>
created_at = <Date 2017-05-31.21:11:25.259>
labels = ['3.7', 'docs']
title = 'Descriptors HowTo: Example on function.__get__ needs update'
updated_at = <Date 2017-06-05.03:10:35.443>
user = 'https://github.com/rmariano'
```

bugs.python.org fields:
```python
activity = <Date 2017-06-05.03:10:35.443>
actor = 'Mariatta'
assignee = 'Mariatta'
closed = True
closed_date = <Date 2017-06-05.03:10:35.444>
closer = 'Mariatta'
components = ['Documentation']
creation = <Date 2017-05-31.21:11:25.259>
creator = 'Mariano Anaya'
dependencies = []
files = []
hgrepos = []
issue_num = 30530
keywords = []
message_count = 6.0
messages = ['294871', '295136', '295154', '295157', '295158', '295159']
nosy_count = 4.0
nosy_names = ['rhettinger', 'docs@python', 'Mariatta', 'Mariano Anaya']
pr_nums = ['1845', '1953', '1954']
priority = 'normal'
resolution = 'fixed'
stage = 'resolved'
status = 'closed'
superseder = None
type = None
url = 'https://bugs.python.org/issue30530'
versions = ['Python 3.5', 'Python 3.6', 'Python 3.7']
```

</p></details>

**Comment by 749c80fe-1c97-4e46-b1dd-74bf62c51f66** [comment]
The example on which tries to create a method from the function's \_\_get__ doesn't work for Python 3.5+. It would give this error:

TypeError: method expected 2 arguments, got 3

The 3rd parameter, needs to be removed.

**Comment by rhettinger** [comment]
Mariatta, feel free to apply and backport once the CLA is signed.

**Comment by Mariatta** [comment]
New changeset 1bced56567335745f91676192fc39c06aab30da9 by Mariatta (Mariano Anaya) in branch 'master':
bpo-30530: Update Descriptor How To Documentation (GH-1845)
https://github.com/python/cpython/commit/1bced56567335745f91676192fc39c06aab30da9

**Comment by Mariatta** [comment]
New changeset 86eb93fae6fc5bf121de815b82697f2bc5bc126c by Mariatta in branch '3.6':
bpo-30530: Update Descriptor How To Documentation (GH-1845) (GH-1953)
https://github.com/python/cpython/commit/86eb93fae6fc5bf121de815b82697f2bc5bc126c

**Comment by Mariatta** [comment]
New changeset 1f04b900861686351977f6a7d51afa5fdc181b82 by Mariatta in branch '3.5':
bpo-30530: Update Descriptor How To Documentation (GH-1845) (GH-1954)
https://github.com/python/cpython/commit/1f04b900861686351977f6a7d51afa5fdc181b82

**Comment by Mariatta** [comment]
Thanks Mariano. I merged your PR and it's been backported to 3.5 and 3.6.
